controllers/fg_handler.go

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import (
+	"github.com/openshift/sandboxed-containers-operator/internal/featuregates"
+)
+
+// Create two enums to represent the state of the feature gates
```

Comment:
two or one enum with two possible values?

Reply:
You are right. one enum type with two values.